src/landing/RegisterForm.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  FormControl, FormHelperText, Input, InputLabel,
+} from '@material-ui/core';
+import './styles.css';
+
+const RegisterForm = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+
+  const validUsername = () => {
```

Comment:
It looks like you have the same validations as the login form. You could export and reuse those instead of writing them again.

Reply:
Might be a good idea to have a separate file to keep validations.

Reply:
Fixed by adding Validations.js